### PR TITLE
prepare for ScalaPB 0.10.10

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

**Comment by raboof** [comment]
PLS BUILD

File: codegen/src/main/scala/akka/grpc/gen/scaladsl/Service.scala [hunk 1]
```diff
@@ -5,9 +5,10 @@
 package akka.grpc.gen.scaladsl
 
 import scala.collection.immutable
-
 import scala.collection.JavaConverters._
 import com.google.protobuf.Descriptors._
+//import com.google.protobuf.compiler.PluginProtos.CodeGeneratorRequest
```
Comment (ennru) [inline codegen/src/main/scala/akka/grpc/gen/scaladsl/Service.scala:10]: Remove.

File: project/plugins.sbt [hunk 2]
```diff
@@ -31,4 +31,4 @@ libraryDependencies += "org.eclipse.jgit" % "org.eclipse.jgit" % "5.10.0.2020120
 // scripted testing
 libraryDependencies += "org.scala-sbt" %% "scripted-plugin" % sbtVersion.value
 
-libraryDependencies += "com.thesamet.scalapb" %% "compilerplugin" % "0.10.9"
+libraryDependencies += "com.thesamet.scalapb" %% "compilerplugin" % "0.10.10"
```
Comment (ennru) [inline project/plugins.sbt:34]: Is this required to be in sync with sbt-protoc?
If so add a note about it.

File: maven-plugin/src/main/scala/akka/grpc/maven/AbstractGenerateMojo.scala [hunk 3]
```diff
@@ -203,11 +203,13 @@ abstract class AbstractGenerateMojo @Inject() (buildContext: BuildContext) exten
...
-      protocbridge.ProtocBridge.run(
-        protocCommand,
+      protocbridge.ProtocBridge.execute(
+        ProtocRunner.fromFunction((args, _) => protocCommand(args)),
         targets,
         Seq(incPath) ++ protocOptions ++ schemas.map(_.getCanonicalPath),
-        pluginFrontend = protocbridge.frontend.PluginFrontend.newInstance)
+        artifact =>
+          throw new RuntimeException(
+            s"The version of sbt-protoc you are using is incompatible with '${artifact}' code generator. Please update sbt-protoc to a version >= 0.99.33"))
```
Comment (ennru) [inline maven-plugin/src/main/scala/akka/grpc/maven/AbstractGenerateMojo.scala:212]: We'll do the sbt-protoc upgrade, will this still apply?

**Review by ennru — APPROVED** [review]
LGTM.

File: project/plugins.sbt (hunk 2 above)
Comment (raboof) [reply project/plugins.sbt:34]: not strictly, I think

File: maven-plugin/src/main/scala/akka/grpc/maven/AbstractGenerateMojo.scala (hunk 3 above)
Comment (raboof) [reply maven-plugin/src/main/scala/akka/grpc/maven/AbstractGenerateMojo.scala:212]: Yes, recent versions of sbt-protoc should not touch this parameter.

(this logic was inside sbt-protoc before, but that constructor became private, so I applied it here as well)

**Pull request merged**